### PR TITLE
Add support for sidecar storybook files

### PR DESCRIPTION
**Pull request by pcardune**

### CODE REVIEW

File: apps/src/templates/GameButtons.story.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Provider} from 'react-redux';
+import {getStore, registerReducers} from '../redux';
+import * as commonReducers from '../redux/commonReducers';
+import GameButtons, {RunButton, ResetButton} from './GameButtons';
+
+registerReducers(commonReducers);
+
+
```
Comment (bcjordan) [inline apps/src/templates/GameButtons.story.jsx:9]: Nit: bonus `\n`

**Comment by Bjvanminnen** [comment]
Should we move all stories to sidecars in this PR?

**Comment by caleybrock** [comment]
same questions as above. why `pcardune-webpack-bundle-images` instead of staging-next? and are we okay with supporting both sidecar and in file storybooks?

**Comment by pcardune** [comment]
yeah, probably best to move everything to sidecars in this PR. This PR happens to depend on the PR for the other branch, so I set the target to that to avoid having those commits show up in this PR as well.  I will move the rest of the storybook code into sidecar files and update this PR.

**Comment by Bjvanminnen** [comment]
Sounds like this PR might have been opened in a rush... :-P

**Pull request merged**